app/templates/complete.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="exit-text text-center">
+  <p>{{t 'previouslogin.line1'}}</p>
```

Comment:
The email address of who to contact is in the consent form. In this case, they can't actually see that page again. But, my understanding is that the local ISP coordinator is the person who gave them their login info so they should know who to contact.